ee/clickhouse/queries/funnels/base.py

```diff
@@ -420,18 +416,14 @@ def _get_breakdown_conditions(self) -> str:
             first_entity = self._filter.entities[0]
 
             values = []
-            if self._filter.breakdown_type == "person":
-                values = get_breakdown_person_prop_values(
+            if self._filter.breakdown_type in ["person", "event"]:
```

Comment:
I'm guessing not relevant as they weren't handled before, but can `breakdown_type` be anything else where we should do something, like "cohort" mentions before.

Reply:
I think this condition isn't needed because in line 235, the breakdown_type == "cohort" is checked and `_get_breakdown_conditions` is only called if breakdown_type != "cohort" which leaves it only possibly equal to `person` or `event`